spec/lib/mcb/courses_editor_spec.rb

```diff
@@ -207,6 +211,30 @@ def run_editor(*input_cmds)
         end
       end
 
+      describe "(subjects)" do
+        it "attaches new subjects" do
+          expect { run_editor("edit subjects", "[ ] Mathematics", "continue", "exit") }.
+            to change { course.subjects.reload.sort_by(&:subject_name) }.
+            from([biology, secondary]).to([biology, mathematics, secondary])
+        end
+
+        it "removes existing subjects" do
+          expect { run_editor("edit subjects", "[x] Biology", "continue", "exit") }.
+            to change { course.subjects.reload.sort_by(&:subject_name) }.
+            from([biology, secondary]).to([secondary])
+        end
+
+        context "when more than 1 course is being edited" do
+          let(:another_course) { create(:course, provider: provider) }
+          let(:course_codes) { [course_code, another_course.course_code] }
+
+          it "does not allow editing subjects" do
+            output, = run_editor("exit")
```

Comment:
what's the comma for after `output`?

Reply:
The `run_editor` method returns a list with two elements. Since I don't use the second element, I threw it away by initially having `output, _` (which in my opinion is a bit clearer) but Rubocop complained that it should be `output,`.